lxc/console.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/gorilla/websocket"
 	"github.com/spf13/cobra"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```

Comment:
Please can you lose the "lxd" here thanks

Reply:
Done